fs.go

```diff
@@ -39,21 +39,14 @@ type Filesystem interface {
 	MkdirAll(filename string, perm os.FileMode) error
 	Join(elem ...string) string
 	Dir(path string) Filesystem
-	Base() string
-}
-
-// Symlinker is a Filesystem with support for creating symlinks.
-type Symlinker interface {
```

Comment:
We have go-billy-siva implementation which might need adjustments to support it, or to return a `unsupported feature error`. Note that this will also break borges again until we update the whole thing, but we can solve that by using vendoring there.